Library/Homebrew/cask/lib/hbc/audit.rb

```diff
@@ -48,6 +49,22 @@ def summary_header
 
     private
 
+    def check_single_uninstall_zap
+      odebug "Auditing single uninstall_* and zap stanzas"
+
+      uninstalls = cask.artifacts.find_all { |k| k.is_a?(Hbc::Artifact::Uninstall) }
```

Comment:
Instead of `#find_all`, use `if cask.artifacts.any? { |k| …  }`.

Reply:
`any?` only lets me check the stanza type, it doesn't allow me to count them. Perhaps you meant `count { |k| ...}`?

Reply:
Ah right, I had an error in thinking here. Yes, `count` sounds good.

Reply:
Done, fix incoming too. I'm writing the tests now.

Reply:
Done, fix incoming.